contributing_to_docs/doc_guidelines.adoc

```diff
@@ -122,6 +114,66 @@ To use a discrete heading, just add `[discrete]` to the line before your unique
 == Managing authorization policies
 ----
 
+== Anchoring titles and section headings
+
+All titles and section headings must have a anchor. The anchor ID must be similar to the title or section heading.
+
+*Example anchor in an assembly file*
+----
+[id="understanding-the-monitoring-stack"]
+= Understanding the monitoring stack
+----
+
+[NOTE]
+====
+Do not include line spaces between the anchor and the section title.
+====
+
+You must add the `{context}` variable to the end of anchor IDs in module files. When called, the `{context}` variable is resolved into the value declared in the `:context:` attribute in the corresponding assembly file. This enables cross-referencing to module IDs in context to a specific assembly and is useful when a module is included in multiple assemblies.
+
+*Example anchor in a module file*
+----
+[id="default-monitoring-components_{context}"]
+= Default monitoring components
+----
+
+[NOTE]
+====
+The `{context}` variable must be preceded by an underscore (`_`) when declared in an anchor ID.
+====
+
+=== Creating unique anchor IDs for 'Prerequisites', 'Additional resources', and 'Next steps' titles
+
+Use unique IDs for *Prerequisites*, *Additional resources*, and *Next steps* titles. You can append `-prerequisites`, `-additional-resources`, or `-next-steps` to a unique string that describes the assembly topic.
```

Comment:
I know you have the note later on, but maybe add "in assemblies" to make it clear up here that it doesn't apply to these sections in the modules.

```
Use unique IDs for *Prerequisites*, *Additional resources*, and *Next steps* titles in assemblies.
```

Reply:
Thank you, that's a great suggestion. I have now resolved this.